### ISSUE TITLE
[RFE] Bump Harvester node driver to v0.6.1

### ISSUE THREAD

**Issue by futuretea** [rancher/rancher#40173]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Bump Harvester node driver to v0.6.1 to suppot multi-nic and multi-disk

**Describe alternatives you've considered**

**Additional context**

Releated Harvester issue:  https://github.com/harvester/harvester/issues/2426

**Comment by khushboo-rancher** [comment]
@TachunLin Could you please help with the testing?

**Comment by TachunLin** [comment]
Verified **fixed** on Rancher `v2.7-head` and `v2.6-head` with Harvester `v1.1.2-rc1`. Close this issue.

### Result
- [ ] Verified node driver v0.6.1 have already built into v2.7-head and v2.6-head
- [ ] Verified Rancher `v2.7-head` and `v2.6-head` both pass the following RKE1 and RKE2 test cases


#### For RKE2
1. Can create new RKE2 guest cluster with multi-nics and multi-disks correctly 
    ![image](https://user-images.githubusercontent.com/29251855/220022500-3b8b38c9-c3dd-4584-a513-35357d45d821.png)
    
    ![image](https://user-images.githubusercontent.com/29251855/220021265-00bcb40a-7166-4e70-8ca3-d6728b1ae187.png)
    
    ![image](https://user-images.githubusercontent.com/29251855/220022606-afdd3df0-1294-4d3b-a066-246d7dcdb4d5.png)
    
    ![image](https://user-images.githubusercontent.com/29251855/220021332-b93ab78f-9a74-4cb0-94d0-252ac9ffa625.png)

2. Can update RKE2 guest cluster setting correctly
    ![image](https://user-images.githubusercontent.com/29251855/220059382-9b6f08ed-9306-479b-9459-2028cefebf8f.png)

3. Can delete RKE2 guest cluster correctly

#### For RKE1 
  1. Can create new RKE1 guest cluster with multi-nics and multi-disks correctly
![image](https://user-images.githubusercontent.com/29251855/220054495-6f72583a-e883-4f47-8f3c-295830c1dc8e.png)

![image](https://user-images.githubusercontent.com/29251855/220054396-70e929cd-0453-4f30-a39e-4d4cf9580038.png)

![image](https://user-images.githubusercontent.com/29251855/220054441-ab916163-d0f1-48b9-b9d6-983b568fb129.png)

![image](https://user-images.githubusercontent.com/29251855/220054569-a662b66b-8bf9-4530-ab66-2df7e34a1d4d.png)

2. Can delete RKE1 guest cluster correctly


### Test Information
* Test Environment: 1 node Harvester on local kvm machine
* Harvester version: v1.1.2-rc1
* Rancher version: v2.7-head (ab561bf), v2.6-head (5d91d1a)
* RKE2 version: v1.25.9, v1.24.9, v1.24.10


#### RKE2 Verification Steps
1. Import Harvester to Rancher and create cloud credential 
2. Check the default version of Harvester node driver is v0.6.1 
3. Create a new RKE2 guest cluster with new node driver and new UI config 
5. Add two new volume
6. Add two new networks
![image](https://user-images.githubusercontent.com/29251855/220020227-bd0cc5bd-d686-4c9c-b1c9-7483da575c81.png)
 
7. Add network data for multiple-nics in Advanced 
  ```
  network:
    version: 1
    config:
     - type: physical
       name: enp1s0
       subnets:
       - type: dhcp
     - type: physical
       name: enp2s0
       subnets:
       - type: dhcp
  ```
  ![image](https://user-images.githubusercontent.com/29251855/220020427-ba7046b6-685e-4c44-b8f5-b372333daeb1.png)

8. Check can create new RKE2 guest cluster correctly with multi-nics and multi-volumes
9. Edit RKE2 cluster config, change cpu, memory and disk size 
10. Check can update RKE2 guest cluster correctly
11. Delete the RKE2 cluster
12. Check can delete RKE2 guest cluster correctly


#### RKE1 verification steps 
1. Import Harvester to Rancher and create cloud credential 
2. Check the default version of Harvester node driver is v0.6.1 
3. Add two new volume
4. Add two new networks
![image](https://user-images.githubusercontent.com/29251855/220051861-4e936bdc-0fc9-4a16-8932-f846acc91174.png)
5. Create a new RKE1 guest cluster with new node driver and new UI config 
6. Delete the RKE1 cluster
7. Check can delete RKE1 guest cluster correctly
